randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -424,17 +424,17 @@ Extra - room_id: [13]
   * Layers: default
   > Door to Mimic Storage
       All of the following:
+          # TODO: dmg
+          Varia Suit
           Any of the following:
               Space Jump or Can Freeze Enemies
               All of the following:
-                  Jump Extends (Advanced) and Can Single Walljump
+                  Jump Extends (Advanced)
```

Comment:
I feel like if you can SWJ, you might as well do this:
![grafik](https://github.com/user-attachments/assets/cac8d2b8-de87-49d9-ace0-7e035883a539)


I'm like 90% sure that its the same setup as in ridley room or zazabi room.

Reply:
that ledge is one tile too tall to grab
![image](https://github.com/user-attachments/assets/005e04ba-c984-4876-9233-5cc47c9ecd60)
if the ledge was 2x1 like this then It would be possible